contracts/modules/capital/Pool.sol

```diff
@@ -389,7 +448,9 @@ contract Pool is IPool, MasterAwareV2, ReentrancyGuard {
     require(ok, "Pool: Transfer failed");
 
     // Transfer cover assets. Start from 1 (0 is ETH)
-    uint coverAssetsCount = coverAssets.length;
+    uint coverAssetsCount = coverAssets.length;  // substract 1 as ETH is skiped
```

Comment:
there's no substraction here the comment is not relevant imo, it's just ETH skipped at the start